functions/api/image/tokens/[[index]].tsx

```diff
@@ -0,0 +1,143 @@
+/* eslint-disable import/no-unused-modules */
+import { ImageResponse } from '@vercel/og'
+import React from 'react'
+
+import getColor from '../../../utils/getColor'
+import getNetworkLogoUrl from '../../../utils/getNetworkLogoURL'
+import { getImageRequest } from '../../../utils/getRequest'
+import getSetup from '../../../utils/getSetup'
+import getToken from '../../../utils/getToken'
+
+export const onRequest: PagesFunction = async ({ params, request }) => {
+  try {
+    const origin = new URL(request.url).origin
+    const { index } = params
+    const networkName = String(index[0])
+    const tokenAddress = String(index[1])
+
+    const cacheUrl = origin + '/tokens/' + networkName + '/' + tokenAddress
+
+    const data = await getImageRequest(cacheUrl, () => getToken(networkName, tokenAddress, cacheUrl))
+
+    if (!data) {
+      return new Response('Asset not found.', { status: 404 })
+    }
+
+    data.ogImage = data.ogImage ?? origin + '/images/192x192_App_Icon.png'
+    data.name = data.name ?? 'Token'
+    data.symbol = data.symbol ?? 'UNK'
```

Comment:
This defaulting belongs in getToken I think

Reply:
<img width="654" alt="Screenshot 2023-08-09 at 2 40 19 PM" src="https://github.com/Uniswap/interface/assets/35351983/c24a08d1-62c6-4840-963a-61c31d1b7821">

how would i bypass this then?

Reply:
discussed offline